examples/meta/src/multiclass/lmnn.sg.in

```diff
@@ -0,0 +1,27 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
+File f_feats_test = read_csv("@SHOGUN_DATA@/fm_test_real.dat")
+File f_labels_train = read_csv("@SHOGUN_DATA@/label_train_multiclass.dat")
+
+#![create_features]
+Features features_train = create_features(f_feats_train)
+Features features_test = create_features(f_feats_test)
+Labels labels_train = create_labels(f_labels_train)
+#![create_features]
+
+#![choose_distance]
+int k = 3
+LMNN lmnn(features_train, labels_train, k)
```

Comment:
is there no factory for this yet?

Reply:
it seems like that there is not factory for `LMNN`.
https://github.com/shogun-toolbox/shogun/blob/c964c9d1aab4bc1cf9133baf14d3bd5b96ba42de/src/shogun/metric/LMNN.h#L32

Reply:
ok that is fine for now